storage/storage/src/main/java/com/webank/wedatasphere/linkis/storage/excel/ExcelStorageReader.java

```diff
@@ -39,7 +39,7 @@
         } else {
             res = XlsxUtils.getBasicInfo(in, file);
         }
-        if(res == null && res.size() < 2){
+        if(res.size() < 2){
```

Comment:
Should check if null in case of nullpoint exception

Reply:
In fact, it would't be null